charts/rmd/values.yaml

```
@@ -0,0 +1,114 @@
+# Default values for rmd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+rmd:
+  # rmd.replicaCount -- (int) Number of Rmd replicas
+  replicaCount: 1
+  image:
+    # rmd.image.repository -- Rmd image name
+    repository: wiktorkepczynski/my_rmd
+     # Specify a tag to override which version of rmd to deploy.
+     # If no tag is specified the appVersion from Chart.yaml is used as tag.
+     # tag:
```

document defaults using helm-docs.
```suggestion
     # rmd.image.tag -- Specify a tag to override which version of rmd to deploy.
     # @default -- appVersion from Chart.yaml
     tag: ~
```